docs/faststore/docs/headless-cms/multiple-page-template.mdx

```diff
@@ -0,0 +1,151 @@
+
+## Template hierarchy logic
+
+Both PDPs and PLPs use a similar logic for selecting the appropriate template to render. This logic prioritizes templates based on various criteria, starting with the most specific and moving to more generic templates, like department or generic PDP/PLP.
+
+Here's the difference between Product Details and Product Listing Page template hierarchy:
+
+| Feature | PDP hierarchy logic | PLP hierarchy logic |
+| ---------- | ---------------------------- | ---------------------------- |
+| Matching Criteria | Slug, Subcategory, Category, Department, Generic. | Slug (subcategory, category, department implied), Generic. |
+| Reliance on Product Information | Considers product details (subcategory, category, department). | Ignores product details beyond the slug. |
+### Product Details Page (PDP) template selection criteria
+
+The template selection process follows a specific hierarchy, starting with the most specific template and progressing towards a generic one if no exact match is found.
+
+FastStore checks for templates in the following order:
+
+```mermaid
+graph TB
+A[Finding the Product Page] --> B{Product Slug match?}
+B --> C{Yes} --> D(Use matching template)
+B --> E{No} --> F{Product Subcategory match?}
+F --> G{Yes} --> H(Use matching template)
+F --> I{No} --> J{Product Category match?}
+J --> K{Yes} --> L(Use matching template)
+J --> M{No} --> N{Product Department match?}
+N --> O{Yes} --> P(Use matching template)
+N --> Q{No} --> R{Use generic template}
+```
+The system first searches for a template named after the product slug. For example, if the product slug is `/blue-shirts`, FastStore will look for a template named `/blue-shirts`. 
+
+1. If no template matches the slug, FastStore searches for a template based on the product subcategory. For example, if the subcategory is `summer clothing`, the system might look for `/clothing/shirts/summer-clothing/`. 
+
+2. Following the subcategory, the system checks for a template matching the product category. For example, if the category is `shirts`, the system might look for `/clothing/shirts/`.
+
+3. If no match is found at the category level, FastStore searches for a template based on the product department. For example, if the department is `clothing`, the system might look for `/clothing/`.
+
+4. If none of the above criteria match, the generic Product Page template is used. This template has an empty Template section value in the settings.
+
+![pdp-page-list](https://vtexhelp.vtexassets.com/assets/docs/src/pdplist___d6f00d8bcfc23aecc477c97d487eebca.png)
+
+### Product Listing Page (PLP) template selection criteria
+
+PLPs adhere to the following template selection logic:
+
+1. FastStore prioritizes templates based on the slug information. This can include the combined department and category (department/category) or just the department (department).
+
+2. If no slug match is found, the generic PLP template is used. |
```

Comment:
```suggestion
1. FastStore prioritizes templates based on the slug information. This can include the combined department and category (department/category) or just the department (department).
2. If no slug match is found, the generic PLP template is used.
```